clang/lib/Serialization/ASTReader.cpp

```diff
@@ -7944,9 +7944,13 @@ void ASTReader::PrintStats() {
   std::fprintf(stderr, "*** AST File Statistics:\n");
 
   unsigned NumTypesLoaded =
-      TypesLoaded.size() - llvm::count(TypesLoaded, QualType());
+      TypesLoaded.size() - std::count(TypesLoaded.materialisedBegin(),
```

Comment:
Does the PagedVector not work with `llvm::count`?

Reply:
Now it does.